src/components/ReadingPage/ReadingPage.js

```diff
@@ -1,37 +1,51 @@
-import React, { Component } from 'react';
-import "./ReadingPage.css"
+import React, { Component } from "react";
+import "./ReadingPage.css";
 
 class ReadingPage extends Component {
-    constructor(props){
-        super(props);
-        this.state = {
-          responseReading: {},
-          currentZodiac: props.currentZodiac
-        }
-    }
+  constructor(props) {
+    super(props);
+    this.state = {
+      responseReading: {},
+      currentZodiac: props.currentZodiac,
+    };
+  }
 
-    componentDidMount () {
-        const URL = 'https://aztro.sameerkumar.website/?sign=aries&day=today';
-        fetch(URL, {
-            method: 'POST'
-        }).then(response => response.json())
-        .then(json => { this.setState({responseReading: json}); });
-    }
+  componentDidMount() {
+    const URL = `https://aztro.sameerkumar.website/?sign=${this.state.currentZodiac}&day=today`;
+    fetch(URL, {
+      method: "POST",
+    })
+      .then((response) => response.json())
+      .then((json) => {
+        this.setState({ responseReading: json });
+      });
+  }
 
-    render() {
-        return (
-          <div>
-              Current Date: {this.state.responseReading.current_date} <br />
-              Compatibility: {this.state.responseReading.compatibility} <br />
-              Lucky Number: {this.state.responseReading.lucky_number} <br />
-              Lucky Time: {this.state.responseReading.lucky_time} <br />
-              Color: {this.state.responseReading.color} <br />
-              Date Range: {this.state.responseReading.date_range} <br />
-              Mood: {this.state.responseReading.mood} <br />
-              Description: {this.state.responseReading.description} <br />
-          </div>
-        );
-    }
+  render() {
+    return (
+      <div>
```

Comment:
It looks to me like this div could use more structure to lay out this info. Have you considered doing something along the lines of wrapping each key and interpolation in a <p> tag (or whatever makes the most sense semantically) then using flex styling to lay them out in a column instead of using the <br /> tag? It would also help with screen readers!